dashboards/google-cloud-composer/README.md

```diff
@@ -0,0 +1,33 @@
+## Cloud Composer Monitoring
```

Comment:
Let's standardize to the same format as the other READMEs in this repo for consistency, e.g.
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-cloud-functions/README.md
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-filestore/README.md
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-cloudsql/README.md

- One notes section
- Since there is only one dashboard, one table linking to the relevant .json
- Dashboard description matches what is in metadata.yaml

Reply:
done!